site/versions/master/docs/tutorial/android-app.md

```diff
@@ -98,6 +109,9 @@ the API levels to the same value for the SDK and NDK.
 [This web page](https://developer.android.com/ndk/guides/stable_apis.html)
 contains a map from Android releases to NDK-supported API levels.
 
+Similar to `android_sdk_repository`, the path to the ndk can be explicitly
+specified with a `path` parameter.
```

Comment:
Please add something along the lines of "or read from the $ANDROID_NDK_HOME environment variable if the `path` attribute is not set", because I don't think we mention the actual environment variable to set anywhere else in this tutorial.